geometry_msgs/Point pose1
geometry_msgs/Point pose2
float32[] covariance